### ISSUE TITLE
[JustMobHeads 1.18.2 and possibly up] Issues wtih Supplimentary's Skull Piles 

### ISSUE THREAD

**Issue by Shibva** [ricksouth/serilum-mc-mods#1694]
## **Information**
Minecraft version: 1.18.2 (and may be in higher versions as well)
Modloader: Forge 
Forge version: 40.1.93
Environment: Singleplayer AND Multiplayer

Mod name: Just Mob Heads
Mod version: 7.1

## **Description**
This issue is a mix between a feature this mod seems to break wtih another mod in terms of functionality and a fatal issue (crash) when its used wtih a pickerang [quark] to "aggravate" the existing issue into a crash.

This report is made to document and inform the other end on the issue; to see the inintal report on the originating end see https://github.com/MehVahdJukaar/Supplementaries/issues/507

TLDR: The behavior added in this mod breaks Supplimentary's "Skull Piles" to where breaking them in survival by normal means results in a bugged state where the render ceases however the block broken itself still exists (simular to how a block dosent actualy break when you mine too fast though blocks like netherrack; this is just me describing the bugged blocks behavior acting simular to such)

This issue is further worsened when using something that breaks it like how Quark's Pickerang works atempts to break it; the result causes a fatal error and crashes the game. I have reason to belive that this secondary issue is the same as the first issue but aggravated into a crash.

From my findings, the only way to break these bugged piles is either in Creative or by using "Create" drill block to break it (being the only means ive found in survival to break these piles in their glitched state)

For Context: Supplementary adds a mechanic to where skulls and heads can be stacked on top of one another to form a Skull Pile that can consist of 2 heads per block. These heads in the pile act independent; meaning that the type of head does not need to match (meaning 2 custom player heads or even a skull and a creeper head can be placed wtihin the same space of the block and so forth)

**As of this time I am unsure if this extends past 18.2 but may be likely if the codework is overall the same**

## **Crash report**

https://gist.github.com/Shibva/33b55f25377d726696ad571330f2de5f
(note: this is the same report found on the other bug report made in the linked github space above

**Comment by ricksouth** [comment]
Thanks for opening the issue, and giving the mod a try! Also for the extensive research, appreciate it a lot. This has been fixed in version 7.3 of Just Mob Heads.

https://curseforge.com/minecraft/mc-mods/just-mob-heads/files
